docs/examples/embeddings/jinaai_embeddings.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>The colab link seems pointing to cohereai notebook. Can you please correct it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>